src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populations = [...populationList];
+
+let total = 0;
+
+populations.forEach((population) => {
+  const popText = population.textContent.replace(/,/g, '');
+  const numericPop = Number(popText);
```

Comment:
```suggestion
  const numericPop = +popText;
```
this syntax is shorter)